### PR TITLE
feat: console mode for watch command

### PR DESCRIPTION
**Pull request by robinbryce**
watch command changes
* now a regular command (visible without IKWID)
* terminates after reporting results once
* will poll once by default, and a maximum of 15 times
* pol interval is a minimum of one second
* exit status is 0 if pending logs are found and non zero otherwise
* output is json formated
* url's are reported for the massif and seals

### CODE REVIEW

File: watcher.go [hunk 1]
```diff
@@ -154,34 +172,106 @@ func NewLogWatcherCmd() *cli.Command {
...
+						}
+
+						activity = append(activity, a)
 					}
-					for _, tenant := range c.SortedSealedTenants() {
-						if watchTenants != nil && !watchTenants[tenant] {
-							continue
+
+					if activity != nil {
+						jd, err := json.MarshalIndent(activity, "", "  ")
```
Comment (henry739) [inline watcher.go:202]: Lets rename `jd` to something clearer

File: cfgreader.go [hunk 2]
```diff
@@ -68,13 +70,16 @@ func cfgReader(cmd *CmdCtx, cCtx *cli.Context, forceProdUrl bool) (azblob.Reader
 	}
 
 	if envAuth {
-		reader, err = azblob.NewDev(azblob.NewDevConfigFromEnv(), container)
+		devCfg := azblob.NewDevConfigFromEnv()
```
Comment (henry739) [inline cfgreader.go:73]: We might be able to rename this to indicate that its about privileged / direct access

File: watcher.go (hunk 1 above)
Comment (robinbryce) [reply watcher.go:202]: will do

File: tests/env.go [hunk 3]
```diff
@@ -37,6 +40,8 @@ type TestEnv struct {
 	PublicTenantId       string
 	MerkelogURLPrefix    string
 	PublicKey            string
+
+	UknownTenantId string
```
Comment (honourfish) [inline tests/env.go:44]: Unknown? or Uknown? if the latter what does the U mean? or is it meant to be KnownTenantd?

Comment (robinbryce) [reply tests/env.go:44]: It's meant to be "a tenant uuid which does not match any known tenant" So Unkown

**Pull request merged**